### PR TITLE
Added item spawns to XML files.

### PR DESCRIPTION
**Pull request by NickJohn547745**

### CODE REVIEW

File: Sprint0/Content/Data/GameFile.xml [hunk 1]
```diff
@@ -5,6 +5,49 @@
 	<Setting name="ResolutionHeight">1080</Setting>
   
 	<Setting name="StartLevelId">0</Setting>
+
+	<Setting name="KeyboardBinds">
```
Comment (Will-Blanton) [inline Sprint0/Content/Data/GameFile.xml:9]: Moving the controls to an XML file significantly reduces the code in game. Nice

**Review by Will-Blanton — APPROVED** [review]
Looks good. The object factories seem to simplify spawning enemies/items by room. I ran the code and it seemed to work as well.

File: Sprint0/Classes/LevelConfig.cs [hunk 2]
```diff
@@ -14,15 +14,17 @@ public class LevelConfig
         public List<List<int>> TileIds { get; set; }
         public List<int> DoorIds { get; set; }
         public List<int> Destinations { get; set; }
-        public Dictionary<int, Tuple<Point, int>> Enemies { get; set; }
+        public List<Tuple<int, Point, int>> Enemies { get; set; }
```
Comment (Will-Blanton) [inline Sprint0/Classes/LevelConfig.cs:17]: You can name the items in the tuple, which may be helpful for making the code more readable. Ex: Tuple<int Id, Point pos> (I think something like this)

**Review by natdrog — APPROVED** [review]
Looks good. I know that a few of the items technically don't spawn in the rooms when they are first loaded, but I think it is good to keep them in anyways for testing purposes.

**Review by Awilkes21 — APPROVED** [review]

**Pull request merged**